packages/ui/src/common/hooks/useLocalStorage.ts

```diff
@@ -23,8 +23,8 @@ function setItem(key: string, value: any) {
   }
 }
 
-export const useLocalStorage = <T extends string>(key: string) => {
-  const [value, setValue] = useState<T>(() => getItem(key))
+export const useLocalStorage = <T>(key: string) => {
+  const [value, setValue] = useState<T | undefined>(() => getItem(key))
```

Comment:
Just realized there is no test for this behavior, will add

Reply:
:eyes: AFAICS there's none :wink: